tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -548,19 +550,17 @@ Validate table rows for featured table
     user checks table cell in offset row contains    ${row}    1    1    5,595
     user checks table cell in offset row contains    ${row}    2    1    6,373
 
-    ${row}=    user gets row number with heading    Bolton 004 (E02000987)
-    user checks table heading in offset row contains    ${row}    0    2    2020
+    ${row}=    user gets row number with heading    Bolton 004
+    user checks table heading in offset row contains    ${row}    0    2    2005
 
-    user checks table cell in offset row contains    ${row}    0    1    6,031
+    user checks table cell in offset row contains    ${row}    0    1    8,557
 
-    ${row}=    user gets row number with heading    Bolton 004 (E05010450)
+    ${row}=    user gets row number with heading    Bolton 004
```

Comment:
Same as above, we've got two identical lines: `${row}= user gets row number with heading Bolton 004`